core/bootstrap/src/main/java/org/eclipse/dataspaceconnector/monitor/ConsoleMonitor.java

```diff
@@ -25,10 +25,17 @@
  * Default monitor implementation. Outputs messages to the console.
  */
 public class ConsoleMonitor implements Monitor {
+
+    private static final String SEVERE = "SEVERE";
+    private static final String ERROR = "ERROR";
```

Comment:
What exactly would be the difference between `SEVERE` and `ERROR`? Do we really need both?